sql/src/test/java/io/crate/integrationtests/SetLicenseIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import io.crate.license.LicenseMetaData;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.junit.Test;
+
+import java.util.Locale;
+
+@ESIntegTestCase.ClusterScope()
+public class SetLicenseIntegrationTest extends SQLTransportIntegrationTest {
+
+    private static final String EXPIRATION_DATE = "2020-12-31T01:02:03.123";
+    private static final long EXPIRATION_DATE_IN_MS = 1609376523123L;  // equivalent for EXPIRATION_DATE
+    private static final String ISSUED_TO = "me";
+    private static final String SIGNATURE = "SIGNATURE";
+
+    private static LicenseMetaData createMetaData() {
+        return new LicenseMetaData(EXPIRATION_DATE_IN_MS, ISSUED_TO, SIGNATURE);
+    }
+
+    @Test
+    public void testSetLicenseCheckClusterStateMetaData () {
+        execute(String.format("set license expiryDate='%s', issuedto='%s', signature='%s'", EXPIRATION_DATE, ISSUED_TO, SIGNATURE));
```

Comment:
Please use `set license expiryDate=?, issuedTo=?, signature=?` and use the other `execute` overload that takes arguments. Using `String.format` to inline values is a bad habbit :)